docs/sources/flow/reference/components/otelcol.exporter.jaeger.md

```diff
@@ -1,5 +1,7 @@
 ---
 canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.exporter.jaeger/
+labels:
+  stage: deprecated
```

Comment:
There's no "deprecated" level in our [stability page](https://grafana.com/docs/agent/latest/stability/#stable). Should we add it? Seem slike a good idea to me.

Reply:
It might be worth noting that Agent isn't following the Grafana release lifecycle stages: https://grafana.com/docs/release-life-cycle/  At some point things will have to be updated to align with the Grafana wording.  This might impact how much effort we place in this (possibly?) short-term fix.

That said, it would be good to define all of the stages currently in use.

Reply:
Thank you for the tip! I raised #5213 about this.